engine/src/main/java/org/hibernate/validator/internal/metadata/location/ParameterConstraintLocation.java

```diff
@@ -22,24 +22,24 @@
  */
 public class ParameterConstraintLocation implements ConstraintLocation {
 
-	private final Callable executable;
+	private final Callable callable;
 	private final int index;
 	private final Type typeForValidatorResolution;
 
-	public ParameterConstraintLocation(Callable executable, int index) {
-		this.executable = executable;
+	public ParameterConstraintLocation(Callable callable, int index) {
+		this.callable = callable;
 		this.index = index;
-		this.typeForValidatorResolution = ReflectionHelper.boxedType( executable.getParameterGenericType( index ) );
+		this.typeForValidatorResolution = ReflectionHelper.boxedType( callable.getParameterGenericType( index ) );
 	}
 
 	@Override
 	public Class<?> getDeclaringClass() {
-		return executable.getDeclaringClass();
+		return callable.getDeclaringClass();
 	}
 
 	@Override
 	public Constrainable getMember() {
```

Comment:
Maybe let's rename the method as well? `getConstrainable()`?

Reply:
Done.